### PR TITLE
Split UnifiedPushMessage to hold internal information

### PR DESCRIPTION
**Pull request by idelpivnitskiy**
JIRA issue: https://issues.jboss.org/browse/AGPUSH-1381

@matzew @sebastienblanc I've split `UnifiedPushMessage` and (for the quick fix) I've removed `ipAddress` from test json files. Is it ok? Should I override `UnifiedPushMessage.toStrippedJsonString()`?

### CODE REVIEW

File: push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/InternalUnifiedPushMessage.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package org.jboss.aerogear.unifiedpush.message;
```
Comment (matzew) [inline push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/InternalUnifiedPushMessage.java]: I wonder if we should move this class out of the push/model package.

That API is used on the Java Sender, and there it does not make sense to have these extra information (like IP address, and others).

Perhaps it's good to move it to push/sender module?

Comment (matzew) [inline push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/InternalUnifiedPushMessage.java]: Let's also move the `clientIdentifier` into this class:
https://github.com/idelpivnitskiy/aerogear-unifiedpush-server/blob/AGPUSH-1381/push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedPushMessage.java#L83

**Comment by matzew** [comment]
@idelpivnitskiy @sebastienblanc Once this is in, I will rebase https://github.com/aerogear/aerogear-unifiedpush-server/pull/543 and leverage this new Internal Message class for some of the push message ID etc

**Comment by matzew** [comment]
:+1: awesome, thanks @idelpivnitskiy

**Comment by matzew** [comment]
Landed in 3d749a8e5ba3bd00d0b6decd019664a095cc06e5

**Comment by idelpivnitskiy** [comment]
@matzew added copyright to my IDEA settings :) thanks!

**Pull request closed without merging**